### DISCUSSION TITLE
v-ripple on <q-item> has an error  or a bug from vue?

### DISCUSSION THREAD

**Discussion by ReaganM02** [quasarframework/quasar#13768] [General - Components / Directives / etc] [answered]
Hi, I hope this is the correct forum to post this issue. 

While using `<q-item v-ripple>`, I noticed that there is an error or a bug based on the warning. I made some research but I couldn't find the solution for this.  

<a href="https://ibb.co/TLMcj4G"><img src="https://i.ibb.co/sH9jxK4/Screen-Shot-2022-06-24-at-10-52-54-AM.png" alt="Screen-Shot-2022-06-24-at-10-52-54-AM" border="0"></a>

This is the code I grab from Quasar usage
          <q-list padding>
            <q-item clickable v-ripple>
              <q-item-section avatar>
                <q-icon name="inbox"></q-icon>
              </q-item-section>

              <q-item-section>
                Inbox
              </q-item-section>
            </q-item>

            <q-item active clickable v-ripple>
              <q-item-section avatar>
                <q-icon name="star"></q-icon>
              </q-item-section>

              <q-item-section>
                Star
              </q-item-section>
            </q-item>

            <q-item clickable v-ripple>
              <q-item-section avatar>
                <q-icon name="send"></q-icon>
              </q-item-section>

              <q-item-section>
                Send
              </q-item-section>
            </q-item>

            <q-item clickable v-ripple>
              <q-item-section avatar>
                <q-icon name="drafts"></q-icon>
              </q-item-section>

              <q-item-section>
                Drafts
              </q-item-section>
            </q-item>
          </q-list>
          
          Thank you!

**Comment by MilosPaunovic** [comment, marked as answer]
Check this out:
https://github.com/quasarframework/quasar/issues/13732#issuecomment-1159682150

Solution is to upgrade Quasar to v2.7.3.
